src/com/chess/board/Board.java

```diff
@@ -54,7 +85,7 @@ public static String[][] getIntialBoard() {
         board[7][3] = "K";
         board[7][4] = "Q";
         board[7][5] = "B";
-        board[7][6] = "N";
+        board[5][5] = "N";
```

Comment:
i think you should write the previous unchanged one as intially all pieces will be in its original position.